src/main/java/com/example/demo/services/BidService.java

```diff
@@ -64,13 +64,25 @@ public synchronized BidModel addBid(BidModel bidModel, User user) throws Invalid
 	}
 	
 	//check item creation
+	//check sorting
 	//check to model
 
 	@Override
-	public Collection<BidModel> getBids(int itemId) {
+	public Collection<BidModel> getBids(int itemId,int limit) {
 		Item item=new Item();
 		item.setId(itemId);
-		return bidsRepo.findByItemEquals(item).stream().map(x->x.toModel()).collect(Collectors.toList());
+		List<BidModel> bids= bidsRepo.findByItemEquals(item).stream().map(x->x.toModel())
+				.collect(Collectors.toList());
+		bids.sort(new Comparator<BidModel>() {
+			@Override
+			public int compare(BidModel o1, BidModel o2) {
+				return o2.getAmount().compareTo(o1.getAmount());
+			}
+		});
+		if(limit!=0&&limit<bids.size()) {
```

Comment:
lets use Integer for limit and null value as indicator that limit is not provided